backend/src/models/db/tables.js

```diff
@@ -211,19 +218,13 @@ function initCourseTable(db) {
 
 function initDegreesTable(db) {
     // Prepare query
-
-    // TODO remove me later
-    const mappedDegrees = degreeData.map(({ name, ...rest }) => ({
-        name: name.startsWith('Bachelor of') ? 'B.' + name.split('Bachelor of')[1] : name,
```

Comment:
Does this get dealt with somewhere else though as these names are really long.

Reply:
I'm not sure whatcha mean by 'dealt with'
Maybe we should keep the full name field and also have a shortened name field?

Reply:
Sure. Don't mind. I was thinking we should really remove duplicates like co-op, honours, and remove Bachelor of to be B. and then filter on duplicates.